lib/notification_service/notification_service/server.py

```diff
@@ -113,6 +115,8 @@ def _init_server(self):
                                              port=int(self.config.port))
 
     def start(self, is_block=False):
+        logger.info("Starting notification server with config: {}".format(self.config))
```

Comment:
IMO, it's recommended to log the config file path and the detail of the config.

Reply:
The `NotificationServerRunner` does not know where the log is. When running in the foreground, the log is printing to the stdout.

Reply:
I mean the the path of the notification config, not the log.

Reply:
It has been added in cb190aa